README.md

```diff
@@ -96,12 +97,13 @@ This library deals with three different types of input:
   end
 ```
 
-The most basic functions are `from_str!` and `to_str!` which translate a SIWE message string to a parsed SIWE message and back (respectively). To simplify using the variables from the above example:
+The most basic functions are `parse` and `to_str` which translate a SIWE message string to a parsed SIWE message and back (respectively). To simplify using the variables from the above example:
 
 ```
-iex> parsed = Siwe.from_str!(String.trim(msg))
+iex> {:ok, parsed} = Siwe.from_str!(String.trim(msg))
```

Comment:
```suggestion
iex> {:ok, parsed} = Siwe.parse!(String.trim(msg))
```

Reply:
Fixed this, along with a few other typos.